src/core/path.c

```diff
@@ -138,15 +138,13 @@ QuicPathSetValid(
     Path->IsPeerValidated = TRUE;
     QuicPathSetAllowance(Connection, Path, UINT32_MAX);
 
-    if (Path->IsPeerValidated && Reason == QUIC_PATH_VALID_PATH_RESPONSE) {
+    if (Reason == QUIC_PATH_VALID_PATH_RESPONSE) {
```

Comment:
We don't have any testing for different paths, right?

Reply:
We have some code that tests path changes, that do hit these cases. There were actually some failures in the new code that were hit by these cases.

The reason the extra statement was removed is it was set to TRUE 3 lines above in all cases, so it was a useless check anyway.